docs/add_ops.md

```diff
@@ -38,7 +38,8 @@ PS: IREE is pronounced Eerie, and hence the ghost icon.
 
 ### How to TorchToLinalg
 
-You will need to do 4 things:
+You will need to do 5 things:
+- make sure -DTORCH_MLIR_ENABLE_JIT_IR_IMPORTER=ON is added during build. This is to enable the python file used in `build_tools/update_torch_ods.sh` and `build_tools/update_abstract_interp_lib.sh`
```

Comment:
Can you file an issue and reference it here for having a pure Python version of these?

Reply:
https://github.com/llvm/torch-mlir/issues/3695